src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -159,115 +159,197 @@ public partial interface ITensor<TSelf, T> : System.Collections.Generic.IEnumera
     }
     public static partial class Tensor
     {
-        public static System.Numerics.Tensors.TensorSpan<T> AddInPlace<T>(System.Numerics.Tensors.TensorSpan<T> left, System.Numerics.Tensors.TensorSpan<T> right) where T : System.IEquatable<T>, System.Numerics.IEqualityOperators<T, T, bool>, System.Numerics.IAdditionOperators<T, T, T>, System.Numerics.IAdditiveIdentity<T, T> { throw null; }
-        public static System.Numerics.Tensors.TensorSpan<T> AddInPlace<T>(System.Numerics.Tensors.TensorSpan<T> input, T val) where T : System.IEquatable<T>, System.Numerics.IEqualityOperators<T, T, bool>, System.Numerics.IAdditionOperators<T, T, T>, System.Numerics.IAdditiveIdentity<T, T> { throw null; }
+        public static System.Numerics.Tensors.Tensor<T> AbsInPlace<T>(System.Numerics.Tensors.Tensor<T> input) where T : System.IEquatable<T>, System.Numerics.IEqualityOperators<T, T, bool>, System.Numerics.INumberBase<T> { throw null; }
```

Comment:
All of these "in place" ones could just be operating over `TensorSpan` to avoid needing to have multiple overloads.

The regular ones like `Abs` are a bit more difficult since they need to allocate the destination buffer. So I think we need both `Tensor<T> Abs(ROTensorSpan<T> input)` and `TensorSpan<T> Abs(ROTensorSpan<T> input, TensorSpan<T> destination)`

Reply:
Ah, nevermind, I see it's split between `Tensor` and `TensorSpan`.